server.js

```diff
@@ -47,15 +47,18 @@ app.get("/scrape", function (req, res) {
       result.link = "https://tampabay.com" + $(this).children("a").attr("href");
       result.topic = $(this).find(".sectionbullet").text();
       result.thumb = $(this).children("img").attr("src");
```

Comment:
not attaching to the right html element 
![Screen Shot 2020-04-29 at 9 04 25 PM](https://user-images.githubusercontent.com/16521382/80664962-c03c7280-8a5d-11ea-9c0a-98193de63f02.png)